systemtest/src/main/java/io/strimzi/systemtest/performance/report/TopicOperatorPerformanceReporter.java

```diff
@@ -25,13 +26,34 @@ public class TopicOperatorPerformanceReporter extends BasePerformanceReporter {
     protected Path resolveComponentUseCasePathDir(Path performanceLogDir, String useCaseName, Map<String, Object> performanceAttributes) {
         final String maxBatchSize = performanceAttributes.getOrDefault(PerformanceConstants.TOPIC_OPERATOR_IN_MAX_BATCH_SIZE, "").toString();
         final String maxBatchLingerMs = performanceAttributes.getOrDefault(PerformanceConstants.TOPIC_OPERATOR_IN_MAX_BATCH_LINGER_MS, "").toString();
-        final boolean clientsEnabled = !performanceAttributes.getOrDefault(PerformanceConstants.TOPIC_OPERATOR_IN_NUMBER_OF_CLIENT_INSTANCES, "0").equals(0);
+        final String numberOfTopics = performanceAttributes.getOrDefault(PerformanceConstants.TOPIC_OPERATOR_IN_NUMBER_OF_TOPICS, 0).toString();
+        final boolean clientsEnabled = !performanceAttributes.getOrDefault(PerformanceConstants.TOPIC_OPERATOR_IN_NUMBER_OF_CLIENT_INSTANCES, "0").equals("0");
 
-        // Use the useCaseName to create a directory specific to the current test case (Alice or Bob)
-        final Path topicOperatorUseCasePathDir = performanceLogDir.resolve(useCaseName + "/max-batch-size-" + maxBatchSize + "-max-linger-time-" + maxBatchLingerMs + "-with-clients-" + clientsEnabled);
+        final StringBuilder dirPathBuilder = new StringBuilder();
+        dirPathBuilder.append(useCaseName)
+            .append("/max-batch-size-").append(maxBatchSize)
+            .append("-max-linger-time-").append(maxBatchLingerMs)
+            .append("-with-clients-").append(clientsEnabled);
 
-        LOGGER.info("Resolved performance log directory: {} for use case '{}'. Max batch size: {}, Max linger time: {}, Clients enabled: {}",
-            topicOperatorUseCasePathDir, useCaseName, maxBatchSize, maxBatchLingerMs, clientsEnabled);
+        if (!numberOfTopics.equals("0")) {
+            dirPathBuilder.append("-number-of-topics-").append(numberOfTopics);
+        }
+
+        String processType = null;
+
+        if (useCaseName.equals(PerformanceConstants.TOPIC_OPERATOR_FIXED_SIZE_OF_EVENTS_USE_CASE)) {
+            processType = performanceAttributes.get(PerformanceConstants.TOPIC_OPERATOR_IN_PROCESS_TYPE).toString().toLowerCase(Locale.ROOT);
+
+            // it would be either "topic-concurrent" or "batch-concurrent"
+            dirPathBuilder.append("-process-type-").append(processType);
+        }
+
+        final Path topicOperatorUseCasePathDir = performanceLogDir.resolve(dirPathBuilder.toString());
+
+        LOGGER.info("Resolved performance log directory: {} for use case '{}'. Max batch size: {}, Max linger time: {}, " +
+                "Clients enabled: {}, Number of topics: {}, Process type: {}",
+            topicOperatorUseCasePathDir, useCaseName, maxBatchSize, maxBatchLingerMs, clientsEnabled,
+            numberOfTopics.equals("0") ? "not included" : numberOfTopics, processType == null ? "not included" : processType);
```

Comment:
I don't know, should it be on `info` level?
Also, the name of the directory is self explaining, so maybe printing the full path would be sufficient? That way you would not need the `processType` there and checking if it's null or not.

Reply:
> I don't know, should it be on info level?

I can change it to debug...but I don't see any problem with having it to INFO cause it's only printed once... :)